src/components/PayPalButtons.test.tsx

```diff
@@ -332,18 +332,20 @@ describe("<PayPalButtons />", () => {
         const spyConsoleError = jest
             .spyOn(console, "error")
             .mockImplementation();
-        // eslint-disable-next-line @typescript-eslint/no-non-null-assertion
-        window.paypal!.Buttons = () => {
-            return {
-                close: jest.fn().mockResolvedValue({}),
-                isEligible: jest.fn().mockReturnValue(true),
-                render: jest.fn((element: string | HTMLElement) => {
-                    // simulate adding markup for paypal button
-                    if (typeof element != "string")
-                        element.append(document.createElement("div"));
-                    return Promise.reject("Unknown error");
-                }),
-            };
+        window.paypal = {
+            Buttons() {
+                return {
+                    close: jest.fn().mockResolvedValue({}),
+                    isEligible: jest.fn().mockReturnValue(true),
+                    render: jest.fn((element: string | HTMLElement) => {
```

Comment:
I just noticed that this test is meant to fail and throw a rendering error. Let's remove the conditional check and just error.
```suggestion
                    render: jest.fn((element: string | HTMLElement) => {
                       return Promise.reject("Unknown error");
                    }
```

Reply:
We cannot remove the element coz this line on the render process: https://github.com/paypal/react-paypal-js/blob/fcaced0a17b15ec636b0b86bdf8d2a8f2a4e0983/src/components/PayPalButtons.tsx#L106

Reply:
If the element is empty we omit the error process.

Reply:
Ahh good point. Let's keep it then. Please use curly brackets with the `if` and strict equality with `!==`

Reply:
Enabled rules for those cases in eslintrc.json file.